src/Proto.Actor/EventStream.cs

```diff
@@ -56,9 +56,18 @@ public Subscription<T> Subscribe(Action<T> action)
 
         public void Publish(T msg)
         {
+            var logger = Log.CreateLogger<EventStream>();

```

Comment:
ooh. I see. Does it mean that there is the same problem: https://github.com/AsynkronIT/protoactor-dotnet/blob/dev/src/Proto.Actor/EventStream.cs#L19 ?